### PR TITLE
added benchmarking.nodejs.org setup to www

### PR DESCRIPTION
**Pull request by rvagg**
for https://github.com/nodejs/build/issues/281

this is live on the site now

### CODE REVIEW

**Comment by rvagg** [comment]
ssh keypair is in secrets/build/infra/benchmark_id_rsa & secrets/build/infra/benchmark_id_rsa.pub

**Comment by rvagg** [comment]
As per #348, build ignores the charts/ directory and then it does an rsync from the benchmarking server for the real charts/ directory, i.e. there's a 6-hourly update but it'll also update whenever you push to master on the benchmarking repo.
Sound good @mhdawson?

**Comment by mhdawson** [comment]
The description sounds good.

**Comment by mhdawson** [comment]
Can this be closed now ?

File: setup/www/tasks/site-setup.yaml [hunk 1]
```diff
@@ -41,6 +42,7 @@
     - '*  *    * * *   nodejs  /home/nodejs/check-build-site.sh nodejs'
     - '*  *    * * *   nodejs  /home/nodejs/check-build-site.sh iojs'
     - '*  *    * * *   root    /home/nodejs/cdn-purge.sh'
+    - '* */4   * * *   nodejs  rsync -avz --delete benchmark:charts/ /home/www/benchmarking/charts/'
```
Comment (jbergstroem) [inline setup/www/tasks/site-setup.yaml]: Suggesting you remove `-v` -- no need to be verbose. Perhaps even replace with `-q` unless you want cron output sent to your mail.

File: setup/www/tasks/user.yaml [hunk 2]
```diff
@@ -93,6 +93,16 @@
     - iojs/docs
   tags: user
 
+- name: User | Insert benchmarking SSH config and key to nodejs
+  copy:
+    src: ./{{ item.src }}
+    dest: /home/nodejs/.ssh/{{ item.dest }}
+    mode: 0600
```
Comment (jbergstroem) [inline setup/www/tasks/user.yaml]: `0400`?

**Comment by jbergstroem** [comment]
Added some nitpick! LGTM regardless.

**Comment by mhdawson** [comment]
I assume this should still be landed. @rvagg can you address @jbergstroem comments and then I'll land ?

**Comment by rvagg** [comment]
done and merged, need to fix up my other www PR now that I've discovered this one ..

**Pull request closed without merging**